content/How_To/contribute/prerequisits.md

```diff
@@ -0,0 +1,19 @@
+---
+title:   Pre-Requisits to Contribute to Babylon.js
+image:
+description: Pre-requisits for contributing to Babylon.js.

```

Comment:
Typo? (occurs in multiple places)

Reply:
these will be addressed once we merge to master. @PirateJC FYI